### PR TITLE
Utilities: add missing file header

### PR DESCRIPTION
**Pull request by compnerd**
Add a file copyright header to a newly minted file.

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please test

**Review by artemcm — APPROVED** [review]
Thanks.

File: Sources/SwiftDriver/Utilities/Version.swift [hunk 1]
```diff
@@ -1,3 +1,14 @@
+//===--------------- Version.swift - Version Handling Routines ------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
```
Comment (artemcm) [inline Sources/SwiftDriver/Utilities/Version.swift:5]: 2023 :)

**Comment by compnerd** [comment]
@swift-ci please test

**Pull request merged**